.github/workflows/issues.yml

```diff
@@ -10,6 +10,13 @@ jobs:
   triage-labels:
     runs-on: ubuntu-latest
     steps:
+      - name: If no labels, add to scrum board
+        uses: actions/add-to-project@v0.3.0
+        if: ${{ join(github.event.issue.labels.*.name, ',') == '' }}
+        with:
+          github-token: ${{ secrets.PROJECT_BOARD_ACTIONS_TOKEN }}
+          project-url: https://github.com/orgs/microsoft/projects/145/views/22
```

Comment:
I think it should be added to view 1. The view 22 is this month's view and will deleted

Reply:
Oh, or maybe view 17? https://github.com/orgs/microsoft/projects/145/views/17

Reply:
I was a bit unclear on how columns were handled -- how to get the new items to appear in the board's triage column. Maybe using view 17 is the solution there.